bin/i18n/i18n_script_utils.rb

```diff
@@ -378,6 +387,16 @@ def self.copy_file(file_path, dest_path)
     FileUtils.cp(file_path, dest_path)
   end
 
+  # Moves file
+  #
+  # @param file_path [String] path to the file
+  # @param dest_path [String] destination path
+  def self.move_file(file_path, dest_path)
+    dest_dir = File.extname(dest_path).empty? ? dest_path : File.dirname(dest_path)
+    FileUtils.mkdir_p(dest_dir)
+    FileUtils.mv(file_path, dest_path, force: true)
+  end
+
```

Comment:
I see tests for `write_file` and `copy_file`, but not for `move_file`.

Reply:
https://github.com/code-dot-org/code-dot-org/blob/83f0924e27f42dd3d2c9fec72db07551f9c3c6c0/bin/test/i18n/test_i18n_script_utils.rb#L338